src/components/dev-hub/stories/chart.stories.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import Chart from '../chart';
+
+<Meta title="Chart" />
+
+# Embedded MongoDB Chart
+
+We support embedded MongoDB Charts through the component factory for articles.
+
+<Preview>
+    <Story name="Chart">
+        <Chart
+            nodeData={{
+                options: {
+                    id: '6f921f0c-a1ee-4106-839c-412fad3c64e7',
+                    autorefresh: 100,
```

Comment:
This seems like it's going to be making a lot of network requests when looking at the story.

Reply:
Yeah I bumped this down in the story. The default value is `3600` so shouldn't go off too often now.